### ISSUE TITLE
Unable to respond to message just sent

### ISSUE THREAD

**Issue by VincentJ-Caliopen** [CaliOpen/Caliopen#944]
1/ I sent a message to someone, it works
2/ I then reply to the message : I get "unable to send the message" error

In console I see:
- method : 
```
Request URL:https://alpha.caliopen.org/api/v1/messages
Request Method:POST
```
- payload : 
```json
{
	"body": "test9.1.1",
	"body_is_plain": true,
	"excerpt": "test9…",
	"date": "2018-05-24T15:02:19.720Z",
	"date_insert": "2018-05-24T15:02:19.720Z",
	"date_sort": "2018-05-24T15:02:19.720Z",
	"discussion_id": "dd8a3875-51a6-467c-82f5-65dcccb97c81",
	"external_references": {},
	"identities": [{
		"identifier": "usertest@alpha.caliopen.org",
		"type": "email"
	}],
	"importance_level": 0,
	"is_answered": false,
	"is_draft": true,
	"is_unread": false,
	"is_received": false,
	"message_id": "de6a7db7-a320-4b5a-b029-b986de1cdaab",
	"type": "email",
	"user_id": "934f7364-bbc2-4dd2-91cf-50c3745ed2dd"
}
```

**Comment by sapiens-sapide** [comment]
Stop using the `?discussion_id=` param to fetch last known state of a specific message.

**Comment by iamdey** [comment]
but how ?

I add a message, How can I do that if I want to make sure my related collection is up to date (at least know if the message has been added to the collection) ?

**Comment by sapiens-sapide** [comment]
If you've got a `200` you know that message has been created or patched

**Comment by iamdey** [comment]
Unfortunately I merged in develop so the fix will be available for the next release (0.11)
